package.json

```diff
@@ -22,31 +22,28 @@
   },
   "devDependencies": {
     "aws-sdk": "~2.2.43",
-    "babel-plugin-feature-flags": "~0.2.0",
+    "babel-plugin-feature-flags": "0.2.0",
```

Comment:
isn't this the only thing you wanted to change?  The rest of the changes seem problematic for a release branch